src/Laravel/TelegramServiceProvider.php

```diff
@@ -48,6 +48,7 @@ public function register()
     {
         $this->registerManager($this->app);
         $this->registerBindings($this->app);
+        $this->commands('telegrambot.commands.webhook');
```

Comment:
I think we should change this to `telegram.bot.commands.webhook`.

Reply:
This is a string. The dots as you know have no technical meaning.
It is a matter of human test to put dots.
I put them.